modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/init.jsp

```diff
@@ -29,10 +29,7 @@ taglib uri="http://liferay.com/tld/util" prefix="liferay-util" %>
 page import="com.liferay.analytics.reports.web.internal.display.context.AnalyticsReportsDisplayContext" %><%@
 page import="com.liferay.analytics.reports.web.internal.util.AnalyticsReportsUtil" %><%@
 page import="com.liferay.portal.kernel.language.LanguageUtil" %><%@
-page import="com.liferay.portal.kernel.util.PortalUtil" %><%@
-page import="com.liferay.portal.kernel.util.SessionClicks" %>
-
-<%@ page import="java.util.Objects" %>
```

Comment:
We made some checks in the JSP, some ifs evaluating if the component should render or not... Now, we render the component always because we need the JS code available. Does this make sense to you?